greedy_methods/dota2_senate.py

```diff
@@ -0,0 +1,52 @@
+from collections import deque
+
+
+def predict_party_victory(senate: str) -> str:
+    """
+    Predict the winning party in the Senate based on the order of actions.
+
+    Args:
+        senate (str): A string representing the order of senators in the Senate.
+        'R' represents a Republican senator, and 'D' represents a Democratic senator.
+
+    Returns:
+        str: The name of the winning party, either "Radiant" or "Dire."
```

Comment:
Please reconcile the different definitions of what `R` and `D` stand for. ;-)
```suggestion
        senate: A string representing the order of senators in the Senate.
        'R' represents a Republican senator, and 'D' represents a Democratic senator.

    Returns:
        The name of the winning party, either "Radiant" or "Dire."
```